module-audio/Audio/AudioCommon.hpp

```diff
@@ -95,7 +95,8 @@ namespace audio
     enum class EventType
     {
         // HW state change notifications
-        JackState,               //!< jack input plugged / unplugged event
+        JackState, //!< jack input plugged / unplugged event
+        MicrophoneState,
```

Comment:
please add a comment as in rest of the events :)

Reply:
Typical "I'm gonna do it later" mistake - added ;)